src/test/java/API/DashboardAPITest.java

```diff
@@ -26,4 +31,204 @@ public void testGetDashboard() throws IOException {
 
         Assert.assertEquals(response.getStatusLine().getStatusCode(), 200, "Response code is not 200");
     }
+
+    @Test
+    public void verifyDashboardName() {
+        RestAssured.port = 8080;
+        RestAssured.baseURI = "http://localhost";
+        RestAssured.basePath = "/api/v1/SUPERADMIN_PERSONAL/dashboard";
```

Comment:
This part can be moved to before method, in tests when different value are required it can be assigned. e.g.: `RestAssured.basePath = "/api/v1/SUPERADMIN_PERSONAL/dashboard/3000/add"` in `updateDashboardWithWrongDahboardID`.